projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
```

Comment:
In this, and all, file(s), use two spaces per level of indentation. Convert tabs to spaces.